### PR TITLE
63 avg dam features

### PR DESCRIPTION
**Pull request by cstillwellusgs**

### CODE REVIEW

File: 1_fetch/src/get_sb_data.R [hunk 1]
```diff
@@ -214,6 +214,26 @@ download_children <-function(sites, sb_table_reduced, dldir, workdir, outdir, ou
   return(return_df)
 }
 
+calc_avg_dams <- function(sb_data) {
+  sb_data <- read_csv(sb_data, show_col_types = FALSE) %>%
+    suppressMessages()
+  dams <- sb_data %>%
+    select(COMID, contains("NDAMS"), contains("STORAGE"), contains("MAJOR")) %>%
```
Comment (jds485) [inline 1_fetch/src/get_sb_data.R:221]:
```suggestion
    select(COMID, contains("NDAMS"), contains("NID"), contains("STORAGE"), contains("MAJOR")) %>%
```

Comment (jds485) [reply 1_fetch/src/get_sb_data.R:221]: Not sure if anything else will need to change with this additional variable

**Review by jds485 — COMMENTED** [review]
4th dam variable

File: 1_fetch/src/get_sb_data.R (hunk 1 above)
Comment (jds485) [reply 1_fetch/src/get_sb_data.R:221]: Nevermind, Charlie noted that STORAGE captures both NID_STORAGE and TOTAL_STORAGE

**Review by jds485 — APPROVED** [review]
Looks good! It is surprising that some of these minimally altered reaches have >400 upstream dams (up to 20 major dams). I think we should check with Ken and Ecoflows if they used CAT or TOT alteration attributes to screen these reaches

**Comment by cstillwellusgs** [comment]
Was just discussing dams on another project. I'm guessing there was some normalization by drainage area to determine 'altered' (I guess a traditional threshold is 103 acre-feet per square mile). Would definitely be curious to learn the procedure for Gages2.1.

**Pull request merged**